<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_aruco_creator</class>
 <widget class="QWidget" name="widget_aruco_creator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QSlider" name="horizontalSliderMarkerSize">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxMarkerID">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxDictionary"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxMarkerSize">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dictionary</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Marker ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="horizontalSliderMarkerID">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Marker side pixel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Marker border bit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxMarkerBorderBit"/>
     </item>
     <item row="3" column="2">
      <widget class="QSlider" name="horizontalSliderMarkerBorderBit">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelImage">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSaveAs">
     <property name="text">
      <string>Save as</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>352</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxMarkerID</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderMarkerID</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMarkerID</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxMarkerID</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMarkerSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderMarkerSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMarkerSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxMarkerSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMarkerBorderBit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderMarkerBorderBit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMarkerBorderBit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxMarkerBorderBit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
